seeders/usersSeeder.ts

```diff
@@ -33,10 +33,14 @@ const seedUsers = async (prisma: PrismaClient): Promise<void> => {
             mode: "age",
           }),
           city: faker.address.cityName(),
+          contact_email: random(0, 1) === 1 ? true : false,
+          contact_sms: random(0, 1) === 1 ? true : false,
+          contact_wpp: random(0, 1) === 1 ? true : false,
```

Comment:
As I mentioned in an above PR:
```js
contact_wpp: random(0, 1) === 1
```
The strict comparision brings the boolean value.

Reply:
resolved